code/unit_tests/icon_tests.dm

```diff
@@ -19,7 +19,7 @@
 		for(var/sprite_accessory_type in subtypesof(sprite_accessory_main_type))
 			var/failed = FALSE
 			var/decl/sprite_accessory/sat = sprite_accessory_type
-			var/sat_name = initial(sat.name)
+			var/sat_name = "[initial(sat.name)] ([initial(sat.uid)])"
```

Comment:
It seems like this part of the test is checking if there are any sprite accessories with the same UID and name. Isn't UID checking already done elsewhere? I don't see the purpose of this check if we're grouping by `name (uid)`—grouping by name at least makes sense for display purposes.